README.md

```diff
@@ -1,10 +1,163 @@
+* Extensibility: Happy Path functionality can be extended through various Terraform hooks, custom Terraform modules and Helm Chart customization
+* Reliability: Happy Path is reliable and is production-ready, used by multiple engineering teams at CZI
 
 ### Security
 
 Please note: If you believe you have found a security issue, please responsibly disclose by contacting us at security@chanzuckerberg.com
 
+
+### Repository structure
+
+This project is a monorepo for the components of a Happy ecosystem:
+* `cli/` - the `happy` CLI tool
+* `api/` - the `happy` API server
+* `shared/` - components shared between `api`, `cli` and the `terraform/provider`
+* `hvm/` - the `happy` version manager
+* `terraform/` - a collection of TF modules we use to provision long-lived infrastructure and application stacks
+* `terraform/provider` - `happy` terraform provider
+* `helm-charts/stack` - an experimental helm chart
+* `examples/` - sample applications that illustrate various concepts that `happy` supports, such as sidecars, tasks, multi-service deployments, GPU intense workloads, and so on
+
+
+### Features
+* Manages short-lived infrastructure (we deploy into your compute)
+* Groups services together (we call it a `stack` for co-deployment), each `stack` is isolated, and you can have multiple `stacks` created for the same application.
+* Easily promote changes from lower to higher environments
+* Supports automated deployments through github workflows
+* Has an extensive set of Github workflows
+* Supports both AWS ECS and EKS runtimes, and allows for an easy migration between the two
+* Abstracts out IaC code with the intent that developers should only focus on the application code
+* Supports Linkerd service mesh, mTLS and service-to-service authorization when deployed on EKS with Linkerd installed
+* Plays nicely with `external-dns`, `karpenter`, `cluster-autoscaler`
+* Integrates with Datadog for dashboarding (assuming you have a `datadog` agent deployed into your EKS)
+* Provides service discovery and autoscaling capabilities
+* Supports both amd64 and arm64 architectures
+* Supports metrics collection, health monitoring through healthchecks, and synthetics
+* Supports rolling updates to minimize downtime
+
+### Prerequisites
+
+You will need to have Docker desktop, AWS CLI, and `terraform` installed to use Happy. You can install them and other useful tools by running 
+
+```
+brew tap chanzuckerberg/tap
+brew install awscli helm kubectx kubernetes-cli aws-oidc linkerd jq terraform
+brew install --cask docker
+```
+
+Docker Desktop needs to be running; and aws cli needs to be configured by running `aws configure`, with profiles setup. Make sure `cat ~/.aws/config` does not return an empty string (we assume you already have an AWS account).
+
+In addition to the above, you will need an up and running EKS cluster, that contains a happy environment namespace (it contains a secret called `integration-secret`).
+
+### Install
```

Comment:
maybe we can add a plug for HVM here. even just a link to the README there would be nice

Reply:
We have had this very same discussion with @hspitzley-czi and decided to postpone until HVM has clear instructions on how to install or upgrade `happy`.